diff -rupw curl.orig/Makefile.am curl/Makefile.am
--- curl.orig/Makefile.am	2023-10-05 15:58:50.000000000 +0000
+++ curl/Makefile.am	2023-10-13 21:17:24.000000000 +0000
@@ -159,7 +159,7 @@ CLEANFILES = $(VC10_LIBVCXPROJ) $(VC10_S
 bin_SCRIPTS = curl-config
 
 SUBDIRS = lib src
-DIST_SUBDIRS = $(SUBDIRS) tests packages scripts include docs
+DIST_SUBDIRS = $(SUBDIRS) packages include
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libcurl.pc
@@ -273,8 +273,6 @@ cygwinbin:
 # We extend the standard install with a custom hook:
 install-data-hook:
 	(cd include && $(MAKE) install)
-	(cd docs && $(MAKE) install)
-	(cd docs/libcurl && $(MAKE) install)
 
 # We extend the standard uninstall with a custom hook:
 uninstall-hook:
diff -rupw curl.orig/Makefile.in curl/Makefile.in
--- curl.orig/Makefile.in	2023-10-11 13:40:00.000000000 +0000
+++ curl/Makefile.in	2023-10-13 21:18:20.000000000 +0000
@@ -684,7 +684,7 @@ CLEANFILES = $(VC10_LIBVCXPROJ) $(VC10_S
 
 bin_SCRIPTS = curl-config
 SUBDIRS = lib src
-DIST_SUBDIRS = $(SUBDIRS) tests packages scripts include docs
+DIST_SUBDIRS = $(SUBDIRS) packages include
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libcurl.pc
 LIB_VAUTH_CFILES = \
@@ -1802,8 +1802,6 @@ cygwinbin:
 # We extend the standard install with a custom hook:
 install-data-hook:
 	(cd include && $(MAKE) install)
-	(cd docs && $(MAKE) install)
-	(cd docs/libcurl && $(MAKE) install)
 
 # We extend the standard uninstall with a custom hook:
 uninstall-hook:
